.puppet-lint.rc

```diff
@@ -1 +1,5 @@
 --relative
+--no-parameter_types-check
+--no-parameter_documentation-check
+--no-documentation-check
```

Comment:
I am not aware of such a plugin, I need to check if it actually exists. Does puppet-lint find anything if you don't disable this check?

Reply:
confirmed, without `--no-documentation-check`:
```
manifests//database/read_only_user.pp - WARNING: defined type not documented on line 14 (check: documentation)
```

Reply:
each of those checks must be disabled for puppet-lint to pass.

Reply:
lets keep this disabled here. But I need to dig a bit through puppet-lint, I've the feeling parameter and parameter_documentation are a bit redundant :D

Reply:
I tested them all independently. I'm sure there's overlap, but something leaked through every time.